stable/jenkins/values.yaml

```diff
@@ -15,6 +15,14 @@ Master:
   # AdminPassword: <defaults to random>
   Cpu: "200m"
   Memory: "256Mi"
+  # Environment variables that get added to the init container (useful for e.g. http_proxy)
+  # InitContainerEnv:
```

Comment:
I think we only need one example env var here in the comments - let's keep `http_proxy` but remove the other two? We can keep all three in the readme for completeness

Reply:
Good point, thanks. Will just leave one example in the comments.
Regarding the Init Container, I think it's good practice to have all the initial setup in an Init Container, so the main container can be as vanilla as possible.